eppo_client/bandit.py

```diff
@@ -160,8 +160,12 @@ def weigh_actions(
         self, action_scores, gamma, probability_floor
     ) -> Dict[str, float]:
         number_of_actions = len(action_scores)
-        best_action = max(action_scores, key=action_scores.get)
-        best_score = action_scores[best_action]
+	# Find the max score
+        best_score = max(action_scores.values())
```

Comment:
Damnit so close to keeping it a one line, but this takes the largest lexicographically ordered key
```suggestion
        max(d, key=lambda x: (d.get(x), x))
```
:(